releases/kubernetes-operator.md

```diff
@@ -0,0 +1,76 @@
+---
+title: CockroachDB Kubernetes Operator releases
+summary: Changelog for the Kubernetes Operator for CockroachDB
+toc: true
+docs_area: releases
+---
+
+The CockroachDB [Kubernetes Operator](/docs/{{site.versions["stable"]}}/kubernetes-overview.html) is the recommended way to configure, deploy, and manage {{ site.data.products.core }} clusters on Kubernetes. The Kubernetes Operator is released on a separate schedule and is versioned independently from CockroachDB.
+
+This page briefly announces releases of the Kubernetes Operator and provides links to more information on GitHub.
+
+In addition to monitoring this page, you can subscribe to be notified about releases to the Kubernetes Operator. Visit [CockroachDB Kubernetes Operator source code repository](https://github.com/cockroachdb/cockroach-operator) and click **Watch**.
+
+{{site.data.alerts.callout_success}}
+If you already use the [Helm](https://helm.sh/) package manager to manage your Kubernetes infrastructure, you can manage {{ site.data.products.core }} clusters by using the [CockroachDB Helm chart](https://github.com/cockroachdb/helm-charts/tree/master/cockroachdb) instead of using the Kubernetes Operator. The Helm chart does not leverage the Kubernetes Operator, and there is no automated way to migrate from one method to the other.
```

Comment:
```suggestion
If you already use the [Helm](https://helm.sh/) package manager to manage your Kubernetes infrastructure, you can manage {{ site.data.products.core }} clusters by using the [CockroachDB Helm chart](https://github.com/cockroachdb/helm-charts/tree/master/cockroachdb) instead of using the Kubernetes Operator. The Helm chart does not use the Kubernetes Operator, and there is no automated way to migrate from one method to the other.
```

Reply:
Possible replacement to simplify the language, if similarly meaningful to you.